webapp/config.py

```diff
@@ -0,0 +1,40 @@
+# webapp config
+from os import environ, path
+
+# importing secret key from environment variable
+basedir = path.abspath(path.dirname(__file__))
+
+class Config:
+    """Base config."""
+    SECRET_KEY = environ.get('SECRET_KEY')
+    STATIC_FOLDER = 'static'
+    TEMPLATES_FOLDER = 'templates'
+    FLASK_SERVER_IP = '0.0.0.0'
+    FLASK_SERVER_PORT = 5010
+    DB_TYPE = 'postgresql'
+    DB_NAME = 'containers'
```

Comment:
What if we want to make non default parameters? What about make a constructor for this class where we will parse configuration parameters from some config of commend line and construct object with this parameters?
Otherwise, if user will want to set a non default parameters he will need to edit a source code file.

Reply:
Resolved